main.py

```
@@ -3,113 +3,165 @@
+            resp = requests.get(NVD_API_URL.format(cve_id), timeout=15)
+            resp.raise_for_status()
+            data = resp.json()
+            metrics = data.get("result", {}).get("CVE_Items", [])
+            if metrics:
+                metrics = metrics[0].get("impact", {})
+                cvss = metrics.get("baseMetricV3", metrics.get("baseMetricV2", {}))
+                score = cvss.get("cvssV3", {}).get("baseScore") or cvss.get("cvssV2", {}).get("baseScore")
+                severity = cvss.get("cvssV3", {}).get("baseSeverity") or cvss.get("cvssV2", {}).get("severity")
+                av = cvss.get("cvssV3", {}).get("attackVector") or cvss.get("cvssV2", {}).get("accessVector")
+                return score, severity, av
+            return None, None, None
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)